Lib/test/test_concurrent_futures/test_wait.py

```diff
@@ -114,9 +114,9 @@ def test_all_completed(self):
 
     def test_timeout(self):
         short_timeout = 0.050
-        long_timeout = short_timeout * 10
+        self.event.clear()
```

Comment:
If you want to avoid this, you could make the base class call `self.event.clear()` in a `setUp(self)` method?

Reply:
It's not really necessary because we create a new event in `setUp(self)`.

I wrote it this way because I thought it made it more clear that the event is unset at the beginning of the test, but I can remove the line if you prefer.

Reply:
Oh yes, we're already creating a new event.... I think we can remove the `event.clear()` call (otherwise I would think that there could be some side effect somewhere)